filer/static/filer/js/addons/focal_point.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
```

Comment:
you actually might want to copy the structure (comments, modular pattern, docs) from cms: https://github.com/divio/django-cms/blob/develop/cms/static/cms/js/modules/cms.modal.js#L1